requirements/local.txt

```diff
@@ -0,0 +1,4 @@
+# Packages required for local development
+-r test.txt
+
+jupyter
```

Comment:
It is for ipython notebook, which I use as a repl locally.  Not strictly required, but that's why it's in local.txt.